README.md

```diff
@@ -561,9 +562,15 @@ settings = {
 app = App(routes=routes, settings=settings)
 ```
 
-Returning a string response from a view will default to using the `text/html`
-content type. You can override this by returning a `Response`, including an
-explicit `Content-Type` header.
+Returning a string response from a view will default to using the `text/json`
+content type. This means you will need to override this so that your HTML views
+use the `text/html` content type in their responses. There are several ways you can do this,
+including:
+
+1. by annotating your handler function so that it uses the `HTMLRenderer` (as shown above)
+2. by returning a `Response`, including an explicit `Content-Type` header (see [Renderers](#renderers) section for an example)
+3. by using the `BEFORE_REQUEST` and `AFTER_REQUEST` hooks, which fire before and after all requests
```

Comment:
Let's drop this last point. Yes, you *could* use them for that purpose, but it's a convoluted way around.

Reply:
Agreed. I wasn't sure if I should include it or not. I assume it would only make sense to use BEFORE/AFTER if you're writing a 100% HTML site...at which point, is apistar the framework you're likely to be using?

Reply:
In that case you'd want to set the default renderer to HTML in the global settings (rather than in annotations)